mlir/lib/Target/Cpp/TranslateToCpp.cpp

```diff
@@ -183,6 +183,12 @@ struct CppEmitter {
   // Returns the textual representation of a subscript operation.
   std::string getSubscriptName(emitc::SubscriptOp op);
 
+  // Returns the textual representation of a member (of object) operation.
+  std::string getOperandAndMemberName(emitc::MemberOp op);
+
+  // Returns the textual representation of a member of pointer operation.
+  std::string getOperandAndMemberName(emitc::MemberOfPtrOp op);
+
```

Comment:
Name is a bit opaque, I'd rename to `createMemberAccess`

Reply:
The name is inspired by `getSubscriptName`, but I wasn't really satisfied either with `getOperandAndMemberName`. I renamed to `createMemberAccess` as suggested.